DigitalLearningSolutions.Data.Tests/DataServices/UserDataServiceTests.cs

```diff
@@ -266,6 +278,30 @@ public void ApproveDelegateUsers_approves_delegate_users()
             }
         }
 
+        [Test]
+        public void RemoveDelegateUser_deletes_delegate_user()

```

Comment:
Should we also write a test for not removing the delegate user if they've got progress?

Reply:
I think we should, better safe than errors in the live DB